web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-accordion.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-kejqil.php, but found phpcs-scan-KEjqiL.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).